<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndMain</class>
 <widget class="QMainWindow" name="WndMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frameTitle">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbSourceIcon">
         <property name="text">
          <string>SRC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ScrolledLabel" name="lbRadioText">
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string> &gt; ESKA &lt; P2</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>179</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lbBluetoothIcon">
         <property name="text">
          <string>BT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbTime">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>10:07</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="mainWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrolledLabel</class>
   <extends>QLabel</extends>
   <header>scrolledlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
